test/integration/transformers.spec.ts

```diff
@@ -33,7 +35,7 @@ describe('transformers', () => {
 			},
 			{
 				slug: autumndbTestUtils.generateRandomSlug({
-					prefix: 'transformer',
+					prefix: 'transformer-1-',
```

Comment:
Why the different transformer slug prefixes?

Reply:
It was useful during developing to differentiate the different contracts that were created on each test.